Index: ../../../../../maps/trunk/world/world_105_115
===================================================================
--- ../../../../../maps/trunk/world/world_105_115	(revision 8132)
+++ ../../../../../maps/trunk/world/world_105_115	(working copy)
@@ -1124,10 +1128,6 @@
 y 36
 elevation 12105
 end
-arch lamppost
-x 2
-y 36
-end
 arch dungeon_magic
 x 2
 y 37
@@ -1569,8 +1569,16 @@
 arch cobblestones2
 x 3
 y 34
-elevation 7225
+elevation 10685
 end
+arch button_lever
+msg
+Pit Test #1
+endmsg
+x 3
+y 34
+connected 60
+end
 arch dungeon_magic
 x 3
 y 35
@@ -1580,6 +1588,11 @@
 y 35
 elevation 10034
 end
+arch button_lever
+x 3
+y 35
+connected 61
+end
 arch dungeon_magic
 x 3
 y 36
@@ -1596,8 +1609,15 @@
 arch cobblestones2
 x 3
 y 37
-elevation 12068
+elevation 12236
 end
+arch manhole_closed_1
+hp 6
+sp 38
+x 3
+y 37
+connected 62
+end
 arch dungeon_magic
 x 3
 y 38
@@ -1605,7 +1625,7 @@
 arch cobblestones2
 x 3
 y 38
-elevation 12236
+elevation 8727
 end
 arch dungeon_magic
 x 3
@@ -2029,8 +2049,15 @@
 arch cobblestones
 x 4
 y 34
-elevation 6917
+elevation 6692
 end
+arch pit_closed
+hp 6
+sp 39
+x 4
+y 34
+connected 60
+end
 arch dungeon_magic
 x 4
 y 35
@@ -2040,6 +2067,13 @@
 y 35
 elevation 9036
 end
+arch pit_open
+hp 6
+sp 39
+x 4
+y 35
+connected 61
+end
 arch dungeon_magic
 x 4
 y 36
@@ -2049,6 +2083,11 @@
 y 36
 elevation 10527
 end
+arch button_handle
+x 4
+y 36
+connected 62
+end
 arch dungeon_magic
 x 4
 y 37
@@ -2056,7 +2095,7 @@
 arch cobblestones2
 x 4
 y 37
-elevation 10853
+elevation 7364
 end
 arch dungeon_magic
 x 4
@@ -2074,7 +2113,7 @@
 arch cobblestones2
 x 4
 y 39
-elevation 10685
+elevation 8727
 end
 arch dungeon_magic
 x 4
@@ -2454,7 +2493,7 @@
 arch cobblestones
 x 5
 y 34
-elevation 6692
+elevation 5919
 end
 arch dungeon_magic
 x 5
@@ -2477,19 +2516,22 @@
 arch cobblestones
 x 5
 y 36
+elevation 7098
+end
+arch dungeon_magic
+x 5
+y 36
+end
+arch cobblestones
+x 5
+y 36
 elevation 9133
 end
 arch sign
-name Central Square of Scorn
-name_pl Central Square of Scorn
 msg
-Read The Signs to the right for
-information on where to go and do.
---oOo--
-WARNING: Magic is not permitted
-within Scorn city limits.
---oOo--
-Please do not litter.
+# This is a "correct" HOLE-based manhole modelled after connect/Hole/pit.arc
+# The animation does not work.  The hole function does work, and is aligned
+# with the manhole graphic on the head tile.
 endmsg
 x 5
 y 36
@@ -2498,10 +2540,10 @@
 x 5
 y 37
 end
-arch cobblestones
+arch cobblestones2
 x 5
 y 37
-elevation 9652
+elevation 7364
 end
 arch dungeon_magic
 x 5
@@ -2519,7 +2561,7 @@
 arch cobblestones2
 x 5
 y 39
-elevation 9827
+elevation 8727
 end
 arch dungeon_magic
 x 5
@@ -2528,7 +2570,7 @@
 arch cobblestones2
 x 5
 y 40
-elevation 9701
+elevation 8727
 end
 arch dungeon_magic
 x 5
@@ -2539,6 +2581,13 @@
 y 41
 elevation 9054
 end
+arch manhole_open_3
+hp 6
+sp 39
+x 5
+y 41
+connected 67
+end
 arch dungeon_magic
 x 5
 y 42
@@ -2912,7 +2961,7 @@
 arch cobblestones
 x 6
 y 34
-elevation 5919
+elevation 6692
 end
 arch dungeon_magic
 x 6
@@ -2923,6 +2972,11 @@
 y 35
 elevation 6146
 end
+arch button_handle
+x 6
+y 35
+connected 63
+end
 arch dungeon_magic
 x 6
 y 36
@@ -2930,7 +2984,7 @@
 arch cobblestones
 x 6
 y 36
-elevation 7098
+elevation 6146
 end
 arch dungeon_magic
 x 6
@@ -2975,8 +3029,13 @@
 arch grass
 x 6
 y 40
-elevation 9094
+elevation 5209
 end
+arch button_handle
+x 6
+y 40
+connected 67
+end
 arch dungeon_magic
 x 6
 y 41
@@ -3350,7 +3409,7 @@
 arch cobblestones2
 x 7
 y 34
-elevation 4624
+elevation 10685
 end
 arch dungeon_magic
 x 7
@@ -3359,8 +3418,20 @@
 arch cobblestones2
 x 7
 y 35
-elevation 3552
+elevation 4054
 end
+arch sign
+msg
+#
+# This animation works, but the x1, y1, x1 y1 pieces have the wrong face, so
+# the player view of the closed manhole is messed up.  The animation takes
+# place at x-1, y-1 instead of at x,y, and the hole is not aligned with the
+# location of the head tile (which is misplaced).
+#
+endmsg
+x 7
+y 35
+end
 arch dungeon_magic
 x 7
 y 36
@@ -3368,7 +3439,7 @@
 arch cobblestones2
 x 7
 y 36
-elevation 3081
+elevation 3046
 end
 arch dungeon_magic
 x 7
@@ -3379,6 +3450,13 @@
 y 37
 elevation 6782
 end
+arch manhole_closed_2
+hp 6
+sp 39
+x 7
+y 37
+connected 63
+end
 arch dungeon_magic
 x 7
 y 38
@@ -3401,11 +3479,20 @@
 x 7
 y 40
 end
-arch brush
+arch grass
 x 7
 y 40
-elevation 8003
+elevation 9094
 end
+arch sign
+msg
+#
+# This animation works with the same symptoms as manhole_closed_3.
+#
+endmsg
+x 7
+y 40
+end
 arch dungeon_magic
 x 7
 y 41
@@ -3808,7 +3895,7 @@
 arch cobblestones2
 x 8
 y 36
-elevation 4054
+elevation 3046
 end
 arch dungeon_magic
 x 8
@@ -3819,24 +3906,6 @@
 y 37
 elevation 6009
 end
-arch sign
-name General Information
-name_pl General Information
-msg
-Safe Places to visit...
-Wanderer Inn (Sleep to save!)
-Pirate Inn (in port area)
-Goth's tavern (near GuardHouse)
-City Hall (west of central square)
-Zoo (Old SOuth road) place to see many monsters
-Patch's house (Wanderer's Way)
-Central Library
-Prison of Scorn
-And all the stores
-endmsg
-x 8
-y 37
-end
 arch dungeon_magic
 x 8
 y 38
@@ -4269,30 +4338,6 @@
 y 37
 elevation 5402
 end
-arch sign
-name Beginner Dungeons
-name_pl Beginner Dungeons
-msg
-Beginner Dungeons:
-Beginners 1&2 (Wanderer's Way)
-Jones' House (River road)
-Old City (Dark Well,  Old South road)
-Drinking Well (just below central square)
-Gork's Grovel. (Below Eastern Gate)
-Undead Church (Old South road)
-Smith's House (Old South road)
-Goth's Tavern Cellar
-Friendly Giant's Tower (west wall)
-Newbie Tower. (River road, east)
-Mad Mage's Tower. (south)
-Dangerous Bungalow (West port area)
-Houser of Porters (near fenced area)
-Fun House (North side of main road)
-Alfalfa's House (River road)
-endmsg
-x 9
-y 37
-end
 arch dungeon_magic
 x 9
 y 38
@@ -4703,6 +4748,11 @@
 y 36
 elevation 1277
 end
+arch button_handle
+x 10
+y 36
+connected 64
+end
 arch dungeon_magic
 x 10
 y 37
@@ -4712,23 +4762,12 @@
 y 37
 elevation 2250
 end
-arch sign
-name Advanced Dungeons
-name_pl Advanced Dungeons
-msg
-Advanced Dungeons:
-Prison Courtyard
-Old City (Gatehouse, secret door,quest)
-Old City (well, next to weapon shop)
-Old City Port (well in Port Area)
-The Arena (Port Area)
-Animal Quarantine (Port Area)
-Haunted House (South East of lake)
-Old Mansion (South of East Gate)
-Puddings Place (near Prison)
-endmsg
+arch manhole_closed_3
+hp 6
+sp 39
 x 10
 y 37
+connected 64
 end
 arch dungeon_magic
 x 10
@@ -5161,6 +5200,23 @@
 y 36
 elevation 1274
 end
+arch sign
+msg
+#
+# These animations work, and look correct in-game, though the hole that
+# things drop through is not aligned with the manhole which is the same
+# as manhole_closed_1 except that the graphic is not corrupted.  All that
+# changed from the above is that the faces for the non-head pieces have
+# been removed.  Gridarta places the head at x,y and places "no face"
+# indications at at x+1,y, x,y+1, and x+1,y+1.
+#
+# Specifying no face instead of blank.111 also works, except that in
+# Gridarta, the non-head pieces are "no face" instead of yellow blanks.
+#
+endmsg
+x 11
+y 36
+end
 arch dungeon_magic
 x 11
 y 37
@@ -5170,21 +5226,6 @@
 y 37
 elevation 4147
 end
-arch sign
-name More Information
-name_pl More Information
-msg
-PLAYER KILLING WILL NOT BE TOLERATED.
-Unless BOTH combatants agree to it.
-Killing Town folk will upset the guards!
---oOo--
-For a lot of adventure information,
-talk to people in Goth's tavern, above
-the Eastern Gate House.
-endmsg
-x 11
-y 37
-end
 arch dungeon_magic
 x 11
 y 38
@@ -6067,7 +6108,7 @@
 arch cobblestones2
 x 13
 y 36
-elevation 4873
+elevation 6564
 end
 arch dungeon_magic
 x 13
@@ -6078,6 +6119,13 @@
 y 37
 elevation 7987
 end
+arch manhole_open_1
+hp 6
+sp 39
+x 13
+y 37
+connected 65
+end
 arch dungeon_magic
 x 13
 y 38
@@ -6497,8 +6545,13 @@
 arch cobblestones2
 x 14
 y 36
-elevation 7445
+elevation 4873
 end
+arch button_handle
+x 14
+y 36
+connected 65
+end
 arch dungeon_magic
 x 14
 y 37
@@ -6945,8 +6998,18 @@
 arch cobblestones2
 x 15
 y 36
-elevation 8327
+elevation 7445
 end
+arch sign
+msg
+#
+# This is a "correct" HOLE-based manhole modelled after connect/Hole/pit.arc
+# The animation does not work.  The hole function does work, and is aligned
+# with the manhole graphic on the head tile.
+endmsg
+x 15
+y 36
+end
 arch dungeon_magic
 x 15
 y 37
@@ -7413,7 +7476,7 @@
 arch cobblestones2
 x 16
 y 36
-elevation 7502
+elevation 2239
 end
 arch dungeon_magic
 x 16
@@ -7422,7 +7485,7 @@
 arch cobblestones2
 x 16
 y 37
-elevation 7984
+elevation 2239
 end
 arch dungeon_magic
 x 16
@@ -7853,7 +7916,7 @@
 arch cobblestones2
 x 17
 y 36
-elevation 5554
+elevation 2239
 end
 arch dungeon_magic
 x 17
@@ -7862,8 +7925,15 @@
 arch cobblestones2
 x 17
 y 37
-elevation 6235
+elevation 7984
 end
+arch manhole_open_2
+hp 6
+sp 39
+x 17
+y 37
+connected 66
+end
 arch dungeon_magic
 x 17
 y 38
@@ -8299,8 +8369,13 @@
 arch cobblestones2
 x 18
 y 36
-elevation 3791
+elevation 7502
 end
+arch button_handle
+x 18
+y 36
+connected 66
+end
 arch dungeon_magic
 x 18
 y 37
@@ -8308,7 +8383,7 @@
 arch cobblestones2
 x 18
 y 37
-elevation 2337
+elevation 6235
 end
 arch dungeon_magic
 x 18
@@ -8746,8 +8821,20 @@
 arch cobblestones2
 x 19
 y 36
-elevation 3167
+elevation 5554
 end
+arch sign
+msg
+#
+# This animation works, but the x1, y1, x1 y1 pieces have the wrong face, so
+# the player view of the closed manhole is messed up.  The animation takes
+# place at x-1, y-1 instead of at x,y, and the hole is not aligned with the
+# location of the head tile (which is misplaced).
+#
+endmsg
+x 19
+y 36
+end
 arch dungeon_magic
 x 19
 y 37
